src/NUnitEngine/nunit.engine/Services/TestAgency.cs

```diff
@@ -200,7 +202,7 @@ private Guid LaunchAgentProcess(TestPackage package)
                 case RuntimeType.Mono:
                     p.StartInfo.FileName = NUnitConfiguration.MonoExePath;
                     string monoOptions = "--runtime=v" + targetRuntime.ClrVersion.ToString(3);
-                    if (enableDebug) monoOptions += " --debug";
+                    if (debugTests || debugAgent) monoOptions += " --debug";
```

Comment:
I am assuming that the --debug causes mono to start in the debugger. Is that correct?